internal/gqlparser/remove-tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+
+# Remove tests from library.
+for f in $(find . -name "*_test.go"); do rm $f; done
```

Comment:
[nit]
```suggestion
find . -name "*_test.go" -delete
```

Reply:
Patched in latest changes. Thanks for noticing the simplification was possible there!